examples/decomposition/plot_ica_vs_pca.py

```diff
@@ -37,7 +37,7 @@
 
 from sklearn.decomposition import PCA, FastICA
```

Comment:
Please consider moving the imports into the cell in which they are first used.

Reply:
Thank you very much for your comments @jsike! Your suggestion makes sense. I have moved the imports to the top of the cell where they are first used.